### PR TITLE
[Messenger] Tweaks for DispatchAfterCurrentBusMiddleware entry

### PR DESCRIPTION
**Pull request by weaverryan**
Just some proofreading changes for #10015!

### CODE REVIEW

File: messenger/message-recorder.rst [hunk 1]
```diff
@@ -1,7 +1,7 @@
 .. index::
     single: Messenger; Record messages; Transaction messages
 
-Transactional Messages: Handle Events After CommandHandler is Done
+Transactional Messages: Handle New Messages After Handling is Done
```
Comment (weaverryan) [inline messenger/message-recorder.rst]: It's a bit tricky: in Messenger, messages are called messages. If you're using Messenger as an event bus, you can of course call them events. I'm mostly using the word "message" to keep consistent with messenger (and not confuse with the EventDispatcher), though I keep "event" wording in the code examples.

File: messenger/message-recorder.rst [hunk 2]
```diff
@@ -33,88 +33,14 @@ Doctrine transaction, in which the user has been created.
...
-                        'middleware' => ['validation', 'doctrine_transaction'],
-                    ],
-                    'messenger.bus.event' => [
-                        'default_middleware' => 'allow_no_handlers',
-                        'middleware' => ['validation', 'doctrine_transaction'],
-                    ],
-                ],
-            ],
-        ]);
-
```
Comment (weaverryan) [inline messenger/message-recorder.rst]: This code block didn't seem relevant anymore. I think it originally came from when `dispatch_after_current_bus` was not going to be included as a default middleware, so you needed to show how it was added to config.

File: messenger/message-recorder.rst [hunk 3]
```diff
@@ -185,8 +111,20 @@ buses. For the example, the middleware must be loaded for both the command and e
...
+never be handled. And if an exception is thrown while sending the welcome email, the Doctrine
+transaction will not be rolled back.
+
 .. note::
 
     If ``WhenUserRegisteredThenSendWelcomeEmail`` throws an exception, that exception
     will be wrapped into a ``DelayedMessageHandlingException``. Using ``DelayedMessageHandlingException::getExceptions``
     will give you all exceptions that are thrown while handing a message with the ``DispatchAfterCurrentBusStamp``.
+
+The  ``dispatch_after_current_bus`` middleware is enabled by default. If you're
```
Comment (OskarStark) [inline messenger/message-recorder.rst]: 2 spaces after “The”

**Comment by javiereguiluz** [comment]
Thanks Ryan.

**Pull request merged**